crates/rover-client/src/headers.rs

```diff
@@ -0,0 +1,32 @@
+use crate::RoverClientError;
+use std::env;
+
+const CONTENT_TYPE: &str = "application/json";
+const CLIENT_NAME: &str = "rover-client";
+
+/// Function for building a [HeaderMap] for making http requests.
+///
+/// Takes a single argument, "api_key"m and returns a [HeaderMap].
+pub fn build(api_key: &str) -> Result<reqwest::header::HeaderMap, RoverClientError> {
+    let mut headers = reqwest::header::HeaderMap::new();
+
+    let content_type =
+        reqwest::header::HeaderValue::from_str(CONTENT_TYPE).expect("is valid header");
```

Comment:
might make this a bit readable to do a `use reqwest::header::HeaderValue` up at the top so you can just use `HeaderValue::from_str` directly here

Reply:
Did that for this and for `HeaderMap` :)